FRW/Services/Manager/WalletManager.swift

```diff
@@ -43,7 +54,7 @@ extension WalletManager {
 
 // MARK: - WalletManager
 
-class WalletManager: ObservableObject {
+class WalletManager: ObservableObject, TokenManager {
```

Comment:
Let's create a new TokenManager class instead of mixing Token and Wallet up.

Reply:
GIven the changes described below, the `TokenManager` does not exist anymore